### PR TITLE
Add gt to mwe conversion

### PR DESCRIPTION
**Pull request by sophiecdubber**
Added a new unit conversion method, from gigatonnes to meters water equivalent. Followed same structure as all other unit conversion methods. Also added corresponding tests.

### CODE REVIEW

**Review by alex-ew — COMMENTED** [review]
Overall a simple and well implemented change. I've made some comments on typing and use of variable names. Happy to discuss

File: glambie/util/unit_conversion_helpers.py [hunk 1]
```diff
@@ -79,14 +79,34 @@ def meters_water_equivalent_to_meters(meters_water_equivalent_list: list, densit
                       density_of_ice_in_gt_per_m3) for i in meters_water_equivalent_list]
 
 
+def gigatonnes_to_meters_water_equivalent(gigatonnes_list: list, rgi_area_km2: float,
```
[the remark this answers is not in the document]
Comment (sophiecdubber) [reply glambie/util/unit_conversion_helpers.py:82]: change made

File: glambie/util/unit_conversion_helpers.py [hunk 2]
```diff
@@ -175,6 +195,26 @@ def mwe2meter(variable_in_mwe: float, density_of_water_in_gt_per_m3: float = 997
     return (variable_in_mwe * density_of_water_in_gt_per_m3) / density_of_ice_in_gt_per_m3
 
 
+def gigatonne2mwe(variable_in_gt: float, rgi_area_km2: float,
```
[the remark this answers is not in the document]
Comment (sophiecdubber) [reply glambie/util/unit_conversion_helpers.py:198]: done

[the remark this answers is not in the document]
Comment (sophiecdubber) [reply glambie/util/unit_conversion_helpers.py:198]: done everywhere

File: glambie/util/unit_conversion_helpers.py [hunk 3]
```diff
@@ -1,186 +1,227 @@
...
-    density_of_ice_in_gt_per_m3 : float, optional
+    density_of_ice : float, optional
         The density of ice in Gt per m3, by default 850
 
     Returns
     -------
     A list of measurements in gigatonnes
     """
-    return [meter2gigatonne(i, rgi_area_km2, density_of_ice_in_gt_per_m3) for i in meters_list]
+    return [_convert_meters_to_gigatonnes(i, area, density_of_ice) for i in variables]
```
Comment (alex-ew) [inline glambie/util/unit_conversion_helpers.py:21]: in retrospect, all internal functions can be called _meters_to_gigatonnes etc. The conversions are implicit from module name

File: glambie/util/unit_conversion_helpers.py [hunk 4]
```diff
@@ -1,186 +1,227 @@
-def meters_to_gigatonnes(meters_list: list, rgi_area_km2: float, density_of_ice_in_gt_per_m3: float = 850) -> list:
+from typing import Iterable
```
Comment (alex-ew) [inline glambie/util/unit_conversion_helpers.py:1]: Module should be renamed to mass_height_conversions.py (clearly summarises what it's doing)

**Review by alex-ew — COMMENTED** [review]
Looking much cleaner. Just a couple more rename suggestions.

**Review by alex-ew — APPROVED** [review]
Thanks for the changes. looking good

**Pull request merged**